lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3855,6 +3855,7 @@ mockProtocolParametersForBalancing = (mockProtocolParameters, nodePParams)
             Just $ Cardano.ExecutionUnits 10000000000 14000000
         , Cardano.protocolParamCollateralPercent = Just 1
         , Cardano.protocolParamMaxCollateralInputs = Just 3
+        , Cardano.protocolParamUTxOCostPerByte = Just 4310
```

Comment:
Suggestion: locate the following protocol parameters together:
```hs
        , Cardano.protocolParamUTxOCostPerWord = Just 34482
        , Cardano.protocolParamUTxOCostPerByte = Just 4310
```

Reply:
Additionally, given that these are commonly-used test parameters, you could just import the values defined here:

https://github.com/input-output-hk/cardano-wallet/blob/1f72c95d081c973e492da739faeee4bdefcfd244/lib/core/src/Cardano/Wallet/Primitive/Types/MinimumUTxO/Gen.hs#L162